tests/src/test/scala/cats/collections/PredicateSpec.scala

```diff
@@ -22,6 +24,16 @@ class PredicateSpec extends CatsSuite {
     checkAll("ContravariantMonoidal[Predicate]", ContravariantMonoidalTests[Predicate].contravariantMonoidal[Int, Int, Int])
   }
 
+  {
+    implicit val eqForPredicateInt: Eq[Predicate[Int]] = new Eq[Predicate[Int]] {
+      val sample = -1 to 1 // need at least 2 elements to distinguish in-between values
```

Comment:
why are just two enough? Why not `(-100 to 100)` or something?

Or, better yet, why not test with `Predicate[Byte]` and enumerate all `256` possibilities to check equality.

Reply:
If the domain is just one element, then every predicate is effectively equivalent to either Empty or Everything depending on whether it accepts the one element.

I don't think we need a 201 or 256 cardinality domain because I tried manually introducing some defects in Predicate and it caught them immediately with the 3 elements.

That being said, it doesn't negatively affect the performance of the testsuite on my machine at all so I'd be ok with either.